rust/src/builders/witness_builder.rs

```diff
@@ -360,43 +360,55 @@ impl TransactionWitnessSetBuilder {
         );
     }
 
-    pub fn build(&self) -> Result<TransactionWitnessSet, JsError> {
+    pub(crate) fn build(&self) -> TransactionWitnessSet {
```

Comment:
I don't think we can make this function `pub(crate)` because we need this to be visible to Rust projects that import CML

Reply:
Updated.